source/Server/Decompilation/ILDecompiler.cs

```diff
@@ -23,6 +23,13 @@ public class ILDecompiler : IDecompiler {
             Argument.NotNull(nameof(streams), streams);
             Argument.NotNull(nameof(codeWriter), codeWriter);
 
+            // Write header
+            codeWriter.WriteLine(
+@".assembly SharpLabAssembly

```

Comment:
Can this be written line-by-line? Mostly because I find easier to read than zero-indent `@"` blocks.
Not critical as I can adjust it.